src/metabase/models/notification.clj

```diff
@@ -99,7 +105,19 @@
    notification-handlers
    k
    #(group-by :notification_handler_id
-              (t2/select :model/NotificationRecipient :notification_handler_id [:in (map :id notification-handlers)]))
+              (let [recipients (t2/select :model/NotificationRecipient :notification_handler_id [:in (map :id notification-handlers)])
```

Comment:
updated this so that hydrate `:recipients` also hydrate all of its sub-info:
- for notification-recipient/user => hydrate the user info
- for notification-recipient/group => hydrate all members info